src/scripts/lib/server/index.js

```diff
@@ -1,10 +1,17 @@
 const express = require('express')
 const logger = require('morgan')
 const Pipeline = require('./pipeline')
+const ServerConfig = require('./utils/server-config')
 
 module.exports = class Server {
 
   constructor(root = __dirname) {
+    this.config = new ServerConfig({
+      'isFirstRun': true,
+      'serverPort': '3000'
+    })
```

Comment:
Would it be safer to move these defaults into ServerConfig itself, perhaps as a `static` constant?

It's currently instantiated here, but also in pipeline without any arguments (`static DATA_DIR = new ServerConfig().db.get('clientData')`) and I'm not sure how that would work with load order.